src/_utils/_git_secret_tools.sh

```diff
@@ -213,7 +213,8 @@ function _gawk_inplace {
 
   _temporary_file
 
-  bash -c "gawk ${parms}" > "$filename"
+  # below doesn't handle directories with spaces correctly
```

Comment:
As my quick research showed there's also an issue described in #135 in this line: https://github.com/sobolevn/git-secret/blob/deae0d1cd53d9cb85f3d34992a1afef866842a27/src/_utils/_git_secret_tools.sh#L617

That's why this error happens:

```
path /Users/sobolev/Desktop/vue_docker/folder with spaces/file.txt
gpg: Note: '--pinentry-mode' is not considered an option
gpg: Note: '--quiet' is not considered an option
usage: gpg [options] --decrypt [filename]
```

Reply:
I'm not sure that's right. I think the error happens because we're trying to pass the parameters as a string instead of as an array.